### PR TITLE
use snowflake icon for shop=frozen_food

### PR DESCRIPTION
**Pull request by matkoniecz**

### CODE REVIEW

**Comment by matkoniecz** [comment]
This one sadly has not worked and I am unsure whether I did something wrong, icon is unavailable or would it work after merge and rebuilding iD.

File: data/presets/shop/frozen_food.json [hunk 1]
```diff
@@ -1,5 +1,5 @@
 {
-    "icon": "maki-shop",
+    "icon": "far-snoflake",
```
Comment (arch0345) [inline data/presets/shop/frozen_food.json:2]:
```suggestion
    "icon": "fas-snowflake",
```

**Review by arch0345 — CHANGES_REQUESTED** [review]

File: data/presets/shop/frozen_food.json (hunk 1 above)
Comment (matkoniecz) [reply data/presets/shop/frozen_food.json:2]: tried fas (solid) prefix in https://github.com/openstreetmap/id-tagging-schema/pull/839 - and failed anyway

Comment (arch0345) [reply data/presets/shop/frozen_food.json:2]: I was trying to point out that it should be `snowflake` instead of `snoflake`

**Comment by matkoniecz** [comment]
https://github.com/openstreetmap/id-tagging-schema/pull/839/commits/0103ae0548c1d22a8d5465fd7b0a354f901e1411 is missing here, probably redoing PR after the other is merged will be the best

**Pull request merged**